test/api_getRobotoff_test.dart

```diff
@@ -1,16 +1,17 @@
 import 'package:flutter_test/flutter_test.dart';
-import 'package:openfoodfacts/model/Insight.dart';
-import 'package:openfoodfacts/model/RobotoffQuestion.dart';
-import 'package:openfoodfacts/model/SpellingCorrections.dart';
+//import 'package:openfoodfacts/model/Insight.dart';
+//import 'package:openfoodfacts/model/RobotoffQuestion.dart';
+//import 'package:openfoodfacts/model/SpellingCorrections.dart';
 import 'package:openfoodfacts/openfoodfacts.dart';
-import 'package:openfoodfacts/utils/QueryType.dart';
-import 'test_constants.dart';
+//import 'package:openfoodfacts/utils/QueryType.dart';
+//import 'test_constants.dart';
+
 
 void main() {
 
   group('$OpenFoodAPIClient get robotoff questions', () {
     test('get questions for Noix de Saint-Jacques EN', () async {
-      RobotoffQuestionResult result =
+      /* RobotoffQuestionResult result =
```

Comment:
I agree that these tests should be skipped. However, rather than commenting them out, I think it is better to use either the `skip` parameter or the `@Skip` annotation, see https://pub.dev/packages/test#skipping-tests

Reply:
Thanks @peterwvj, yeah you have right I will use the skip annotation, it is better and cleaner